### DISCUSSION TITLE
Mobile First

### DISCUSSION THREAD

**Discussion by Cleysonlb** [frontendbr/forum#48] [Perguntas e Respostas]
Fala galera tudo bem?!
Então, sempre tive minhas duvidas sobre o mobile first, vale realmente a pena usar essa "metodologia de desenvolvimento"?
Onde eu acho um conteúdo legal para estudar sobre este assunto?!

**Comment by suissa** [comment]
http://nomadev.com.br/passo-a-passo-como-desenvolver-com-atomic-design-mobile-first-e-stylus/

**Comment by joaopedron** [comment]
Estou pegando firme nos estudos de RWD ultimamente e, antes eu não adotava essa metodologia, mas depois de ler bastante sobre eu estou convencido que mobile first é uma necessidade.

**Comment by Cleysonlb** [comment]
Pois é @joaopedron de um tempo pra cá eu também estou organizando meu workflow e adicionando metodologias e tecnologias, vou começar a estudar mobile first o quanto antes ;)
Vlw @suissa pelo link!!!!

**Comment by eduardojmatos** [comment]
Boa @suissa ! :+1:

**Comment by felipefialho** [comment]
@Cleysonlb, utilizo em todos os projetos e acredito ser algo natural, faz muito mais sentido desenvolver assim.

No começo me enrolava um pouco com MediaQueries, até que comecei a usar a ideia de separa-las por componentes e finalmente, por resolução, como mostro [aqui](http://www.felipefialho.com/blog/2015/otimizando-e-organizando-as-media-queries/#.VsHByfIrKHs), assim você fecha escopos e só carrega o que é necessário em cada resolução.

Mas lembrando que Mobile First transcende o código, o ideal é que toda a arquitetura e design do projeto seja desenvolvido dessa forma.

**Comment by Cleysonlb** [comment]
Boa @LFeh ! Vou implementar ao meu workflow o mobile first, como você disse, faz muito sentido desenvolver começando no mobile! ;)

**Comment by afonsopacifer** [comment]
Juro que não é para gerar polemica rsr
Mas gravei essa semana sobre o assunto, se alguém ai tiver interesse :)

https://www.youtube.com/watch?v=Rrys1PVR5tQ
